adal/automation/WinFormsAutomationApp/WinFormsAutomationApp.csproj

```diff
@@ -103,8 +103,8 @@
     <PackageReference Include="Legacy2CPSWorkaround" Version="1.0.0">
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="Newtonsoft.Json" Version="10.*">
-      <PrivateAssets>All</PrivateAssets>
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2">
```

Comment:
I remember I had to move to 10.* because we were getting NuGet warnings. Please make sure we're not getting NuGet warnings :)